src/stores/filter.js

```diff
@@ -16,15 +11,19 @@ import {
 } from '~/constants/filters'
 import { warn } from '~/utils/console'
 
+import { useSearchStore } from '~/stores/search'
+
 export const useFilterStore = defineStore('filter', () => {
   /** @type {import('../store/types').Filters} */
   const filters = reactive(clonedeep(filterData))
-  const searchType = ref(/** @type {import('~/store/types').SearchType} */ (ALL_MEDIA))
 
   /**
-   * @param {import('../store/types').SearchType} type
+   * Search store state can only be accessed inside a function to prevent circular imports.
```

Comment:
Is it possible to treat the filter store as the source of truth for the search type rather than the search store? Would that allow us to avoid the circular import problem?